src/helix-ui/elements/_hx-alert.less

```diff
@@ -0,0 +1,70 @@
+@import "./_hx-element";
+@import "core/hx-icon";
+
+#close,
+#error,
+#info,
+#success,
+#warning {
+  padding-bottom: 0.875em;
```

Comment:
can we combine into one line? `padding: 0.875em 0;`

Reply:
The shorthand you mention will apply to the top padding, too. It's not a 1:1 replacement.